website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,107 @@
+## How typography is referenced
+
+The construction of typographic styles varies heavily depending on context and tooling. This overview will clarify how HDS typography standards are implemented in different tools.
+
+In Figma, reusable text styles are called “Text Styles” and are categorized and stored in the “text panel” to be used in text-based elements. Text styles directly reference font family, weight, line height, size, and many other text-based properties within a single style and are manually maintained. Some Figma styles include:
+
+- `Display/500/Bold`
+- `Body/300/Regular`
+- `Code/300/Regular`
+
+In code, these styles are constructed differently, using “Tokens.” Tokens are generated from an agnostic JSON file and differ from Figma “Text Styles” because a single token does not reference multiple font properties. Instead, each Token defines the different properties of a single text style. This means that constructing the equivalence of Figma’s Text Style `Display/500/Bold` would utilize the following tokens:
+
+- `font-family: var(--token-typography-display-500-font-family);`
+- `font-size:var(--token-typography-display-500-font-size);`
+- `line-height:var(--token-typography-display-500-line-height);`
+- `font-weight:var(--token-typography-font-weight-bold);`
+
+Rather than calling upon each of these tokens in code every time text is styled, the [Text Component](/components/text) was created to replicate how Figma styles group text-based properties.
+
+## Font families
+
+Helios uses the default system fonts available on each OS. These system fonts are stable and support internationalization, code, tabular data, data visualization, etc. Additionally, they improve performance by reducing the number of resources to be loaded, which is important for large applications like those at HashiCorp.
```

Comment:
To Jorys suggestion, but also maybe shortened and using a more active voice?

```suggestion
In code, Helios uses default system fonts because they are stable, support internationalization, and improve performance by reducing resource load—key for large applications at HashiCorp.
```